kobotoolbox/kpi
# feat(organizations): enable data transfer during batch import via Django Admin

### 🗒️ Checklist

1. [ ] run linter locally
2. [ ] update all related docs (API, README, inline, etc.), if any
3. [ ] draft PR with a title `<type>(<scope>)<!>: <title> TASK-1234`
4. [ ] tag PR: at least `frontend` or `backend` unless it's global
5. [ ] fill in the template below and delete template comments
6. [ ] review thyself: read the diff and repro the preview as written
7. [ ] open PR & confirm that CI passes
8. [ ] request reviewers, if needed
9. [ ] delete this section before merging

### 📣 Summary
Automate user data transfer to the organization during batch imports from the Django Admin.


### 📖 Description
This PR introduces functionality to transfer user data to the organization during batch imports performed via the Django Admin interface. For each user included in the batch, their projects and associated data are automatically transferred to the organization as part of the import process. This enhancement streamlines data management, ensuring consistency and reducing the need for manual transfers.

### 👀 Preview steps

1.  Create several accounts (user_1, user_2, etc...). Let's user_1's organization is org1234.
2. Add a project and data for each data
4. Create a csv which contains 3 columns (`is_admin`, `user`, `organization`)
5. Add at least one line matching one of the accounts previously created (where organization id where you want to move users)

is_admin | user | organization
-- | -- | --
False | user_2 | org1234

6. Go to admin interface in Home › Organizations › Organization users.
7. Click import 
8. Log to organization owner account and see every project there and owned by that owner.
